resolwe/flow/serializers/data.py

```diff
@@ -37,11 +36,11 @@ class DataSerializer(ResolweBaseSerializer):
     process_output_schema = ProjectableJSONField(source='process.output_schema', read_only=True)
     process = ResolweSlugRelatedField(queryset=Process.objects.all())
     descriptor_schema = NestedDescriptorSchemaSerializer(required=False)
-    entity_names = serializers.SerializerMethodField()
-    collection_names = serializers.SerializerMethodField()
+    entity_name = serializers.SerializerMethodField()
```

Comment:
We can probably remove this now in favor of `collection` and `entity` serializers defined below.